src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs

```diff
@@ -415,4 +533,28 @@ private static string GetBaseUrl()
         }
         #endregion
     }
+
+    //Refer to https://stackoverflow.com/a/29772349/5765982
+    public static class HttpClientExtensions
+    {
+        public static async Task<HttpResponseMessage> PatchAsync(this HttpClient client, string requestUri, HttpContent content)
+        {
+            var method = new HttpMethod("PATCH");
+            var request = new HttpRequestMessage(method, new Uri(requestUri))
+            {
+                Content = content
+            };
+
+            HttpResponseMessage response = new HttpResponseMessage();
+            try
+            {
+                response = await client.SendAsync(request);
+            }
+            catch (TaskCanceledException e)
+            {
+            }
+
+            return response;
```

Comment:
Don't use `try...catch` here. The method `UpdateGraphic
`  that call this method have handled the error if any error occurs, this method should throw error normally.

Reply:
Yes, I should throw this exception here even I used `try...catch`.

Reply:
I removed the `try...catch` block